src/cuga/backend/tools_env/registry/registry/api_registry.py

```
@@ -60,6 +65,49 @@ async def start_servers(self):
+            )
+            
+            await self.tool_guard_runtime.initialize()
+            
+            guarded_tools = self.tool_guard_runtime.get_guarded_tools()
+            logger.info(
+                f"✅ ToolGuardRuntime initialized with guards for {len(guarded_tools)} tools"
+            )
+            if guarded_tools:
+                logger.debug(f"   Guarded tools: {', '.join(guarded_tools)}")
+            
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)